src/SkipDK.cpp

```diff
@@ -164,6 +164,10 @@ class AzerothCore_skip_deathknight : public PlayerScript
                 Azerothcore_skip_deathknight_HandleSkip(player);
             }
         }
+		if (sConfigMgr->GetOption<bool>("DeleteGold.Deathknight.Optional.Enable", true))
+		{
+			player->SetMoney(1);
```

Comment:
```suggestion
			player->SetMoney(0);
```
Would this not be preferable?

Reply:
What kind of monster are you?? Releasing them into this world of... warcraft without even 1 copper?
Joke aside I can do it, but it's really 1 copper and then also their gear. It's a few silver in total, which shouldn't be a problem imo.

Reply:
Actually, you might want to set it to `CONFIG_START_HEROIC_PLAYER_MONEY`. That way if someone specifies in their config that they want a DK to start with 20g or something, they'll have that at the end of the skip process.

Reply:
From my testing **CONFIG_START_HEROIC_PLAYER_MONEY**  doesn't resolve the problem. It gives you the 20s before the dk-module does it's job.

worldserver.conf
![grafik](https://github.com/azerothcore/mod-skip-dk-starting-area/assets/23155389/56519137-c1b5-4d08-8da4-7fa833bb1bbe)

Right after log in with new char.
![WoWScrnShot_102523_175036](https://github.com/azerothcore/mod-skip-dk-starting-area/assets/23155389/17c6c100-edd5-44c1-bb33-86726adeac70)

After doing the skip.
![WoWScrnShot_102523_175833](https://github.com/azerothcore/mod-skip-dk-starting-area/assets/23155389/657d4732-5063-41fa-9a8f-b638f912ee60)

Also the same amount with instant skip.

Reply:
Yes, I know what the config normally does, I'm saying instead of using a hard-coded value of 0 or 1 copper, why not use the value in the config for this change?